ec2.py

```diff
@@ -0,0 +1,69 @@
+"""
+module for interacting with EC2
+"""
+
+import boto3
+
+
+class EC2Client(object):
+    """
+    Client for interacting with EC2
+    """
+    def __init__(self):
+        self.client = boto3.client('ec2')
+
+    def describe_instances(self, ec2_ids):
+        """
+        @param ec2_ids: list of ec2 instance ids
+        @return: dictionary of ec2 instance ids to description dicts
+        """
+        instances = {}
+
+        paginator = self.client.get_paginator('describe_instances')
+        for resp in paginator.paginate(DryRun=False, InstanceIds=ec2_ids):
```

Comment:
I'm not exposing it to the user, the boto3 docs don't describe a default, so I'm just being explicit about it.